README.md

```diff
@@ -137,3 +137,23 @@ const browser = await launch();
 // Connect to first browser instead
 const anotherBrowser = await launch({ wsEndpoint: browser.wsEndpoint() });
 ```
+
+## BYOB - Bring Your Own Browser
+
+Essentially the process is as simple as running a chromium-like binary with the
+following flags:
+
+```
+chromium --remote-debugging-port=1337 \
+--headless=new \
+--no-first-run \
+--password-store=basic \
+--use-mock-keychain \
+--hide-scrollbars
+```
+
+Technically, only the first flag is necessary, though I've found that these
+flags generally get the best result. Once your browser process is running,
+connecting to it is as simple as
+
+[Example](/examples/byob.ts) included in the `examples` directory.
```

Comment:
Can we move this out of the examples directory? It's a little annoying to view this on the jsr so I'd prefer to just keep it inline.